Sources/Commands/SwiftTestTool.swift

```diff
@@ -851,6 +905,15 @@ extension SwiftTestTool: ToolName {
     }
 }
 
+fileprivate func clearDirectory(path: AbsolutePath) {
```

Comment:
I wonder if this needs to be a method? Essentially, this is just one line of code `localFileSystem.removeFileTree(path)` -- I think we can inline it where it is needed.